kivy/core/text/_text_sdl2.pyx

```diff
@@ -220,3 +236,40 @@ def _get_fontdescent(container):
 
 def _get_fontascent(container):
     return TTF_FontAscent(_get_font(container))
+
+
+cdef _set_font_direction(str direction):
+    global last_base_direction
+    if direction == last_base_direction:
+        return
+
+    IF USE_HARFBUZZ:
```

Comment:
all the calls being conditional, maybe this one is not necessary? Or maybe the whole method declaration could be conditional?

Reply:
Yeah, is is not necessary, however, it would be weird if a function is conditionally declared and just doesn't exist at all depending on a condition. And long term it we may call the function even if there's no harfbuzz if we don't want to guard the call under an if in multiple places. So it just safer and more consistent to always have the function defined.